webapp/app/templates/components.html

```diff
@@ -52,23 +52,35 @@
 {% endmacro %}
 
 {% macro radio_field(field) -%}
-    {% for subfield in field %}
-        <div class="form-row-center">
-            {{ subfield(class="") }}
-            {{ field_label(subfield, class="col-sm-10 col-form-label") }}
-        </div>
-    {% endfor %}
+    <fieldset>
+        {{ field_label(field) }}
```

Comment:
Just an idea: You could also have a macro field_legend(field). You could then get rid of the conditional in field_label.

Reply:
Yeees..but then I'd have to duplicate the whole function creating - again - two places to change if for example we add more to every field laben/legend (e.g. remove the help button)

Reply:
Makes sense